.github/workflows/build.yml

```diff
@@ -26,5 +26,80 @@ jobs:
     - name: Install MAUI workloads
       run: dotnet workload install maui --version 9.0.100
 
+    - name: Create artifacts directory (Windows)
+      if: runner.os == 'Windows'
+      run: New-Item -ItemType Directory -Path "artifacts" -Force
+      shell: powershell
+
+    - name: Create artifacts directory (Unix)
+      if: runner.os != 'Windows'
+      run: mkdir -p artifacts
+      shell: bash
+
     - name: Build
-      run: dotnet build VibeSomeMauiBro/VibeSomeMauiBro.csproj --configuration ${{ github.event_name == 'pull_request' && 'Debug' || 'Release' }}
+      run: dotnet build VibeSomeMauiBro/VibeSomeMauiBro.csproj --configuration ${{ github.event_name == 'pull_request' && 'Debug' || 'Release' }} --verbosity normal -bl:artifacts/build-${{ matrix.os }}.binlog
```

Comment:
`--configuration ${{ github.event_name == 'pull_request' && 'Debug' || 'Release' }}` needs to be saved in a variable for the entire build.

Then everywhere that says `Release` should use the variable.